Doc/whatsnew/2.7.rst

```diff
@@ -2384,7 +2384,7 @@ Port-Specific Changes: FreeBSD
 -----------------------------------
 
 * FreeBSD 7.1's :const:`SO_SETFIB` constant, used with
-  :func:`~socket.getsockopt`/:func:`~socket.setsockopt` to select an
+  :func:`~socket.socket.getsockopt`/:func:`~socket.socket.setsockopt` to select an
```

Comment:
Same as the above. It is not clear that they are methods (think about reading non-HTML document).

Reply:
I agree here, added